src/main/java/org/jenkinsci/plugins/workflow/actions/ErrorAction.java

```diff
@@ -165,7 +167,8 @@ public String getUrlName() {
      * {@link Throwable#equals} might not be reliable if the program has resumed
      * and stuff is deserialized.
      */
-    private static boolean equals(Throwable t1, Throwable t2) {
+    @Restricted(Beta.class)
+    public static boolean equals(Throwable t1, Throwable t2) {
```

Comment:
For my particular case, I could instead create a new `public static boolean isRethrown(FlowNode node)` method that checks whether the `ErrorAction` on the associated `FlowNode` was also thrown by one of its immediate parents, but exposing `equals` directly does not seem particularly problematic to me and may be useful in other cases.